src/components/coordinator/HeaderOrganization.vue

```diff
@@ -37,8 +37,8 @@ export default defineComponent({
   emits: ['export'],
   setup(props) {
     const countBranches = computed((): number => {
```

Comment:
Better const name `subsidiariesCount`.

Reply:
Fixed with https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/639/commits/494f8270ae7c755d58fec2d7506868959d1da0e1.